src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -409,6 +409,20 @@ PARAM_DEFINE_FLOAT(FW_LND_AIRSPD_SC, 1.3f);
  */
 PARAM_DEFINE_FLOAT(FW_LND_THRTC_SC, 1.0f);
 
+/**
+ * Go around throttle threshold
+ *
+ * The vehicle will go around if throttle position larger than threshold
+ * during approach. it will trigger if throttle position passed from below
+ * to above threshold
```

Comment:
Is there a parameter value that disables this feature?  I think it should be disabled by default.  If a user/integrator wants to use this feature then they can choose the appropriate throttle threshold for their vehicle/system.

Reply:
if you set it 0, it will be disable this function, I will disable it by default